### DISCUSSION TITLE
Subroute complete path

### DISCUSSION THREAD

**Discussion by A81N9** [posva/unplugin-vue-router#480] [Questions and Help] [answered]
Is it possible to provide a configuration item to configure the path of the sub-route as a full path?

Such as:

```
- parent
  - child

{
  path: '/parent',
  children: [
    {
      path: 'child', // Convert to '/parent/child'
      name: '/parent/child',
    },
  ],
}
```

**Comment by PindaPixel** [comment]
Use periods instead of folders. Instead of `parent/child.vue` it becomes `parent.child.vue`.

https://uvr.esm.is/guide/file-based-routing.html#nested-routes-without-nesting-layouts

**Comment by posva** [comment, marked as answer]
Use `definePage()` to change the path of any page
